script/add_ccpo_users.py

```diff
@@ -41,6 +42,10 @@ def _load_yaml(file_):
     config = make_config({"default": {"DEBUG": False}})
     app = make_app(config)
     with app.app_context():
-        ccpo_user_file = sys.argv[1]
-        ccpo_users = _load_yaml(ccpo_user_file)
-        add_ccpo_users(ccpo_users)
+        if sys.argv[1] == "-":
```

Comment:
```suggestion
        if len(sys.argv) == 1:
```

Thinking about this, let's make stdin the "default". This also prevents any issues in the very unlikely scenario that the file `-` exists in the current directory.

Reply:
makes sense. its very unorthodox but do you think it would be better to make it `> 0`

Reply:
`len(sys.argv)` will always be `>0`. `sys.argv[0]` is the name of the program itself. So `== 1` checks that the _only_ argument is the name of the program and that no additional CLI arguments were provided. So we either want `== 1` or `< 2`. And 1 always feels like less of a magic number than 2 :)